### PR TITLE
how-to/monitor: Add monitor item for tidb binlog

### PR DESCRIPTION
**Pull request by july2993**
### What is changed, added or deleted? 
add `Queue Size` and `SQL Query Time`

### What is the related PR or file link(s)? 
https://github.com/pingcap/tidb-ansible/pull/604

### CODE REVIEW

File: dev/how-to/monitor/tidb-binlog.md [hunk 1]
```diff
@@ -30,10 +30,12 @@ category: reference
 | Pump Handle TSO | 记录 Drainer 从各个 Pump 获取到的 binlog 的最大 TSO 对应的时间 | | Pull Binlog QPS by Pump NodeID | Drainer 从每个 Pump 获取 binlog 的 QPS |
 | 95% Binlog Reach Duration By Pump | 记录 binlog 从写入 Pump 到被 Drainer 获取到这个过程的延迟时间 |
 | Error By Type | Drainer 遇到的 error 数量，按照 error 的类型进行统计 |
+| SQL Query Time| Drainer 在下游执行 SQL 的耗时 |
 | Drainer Event | 各种类型 event 的数量，event 包括 ddl、insert、delete、update、flush、savepoint |
 | Execute Time | 在下游执行 SQL 语句或写数据所消耗的时间 |
 | 95% Binlog Size | Drainer 从各个 Pump 获取到 binlog 数据的大小 |
 | DL Job Count | Drainer 处理的 DDL 的数量|
+| Query Size | Drainer 内部工作队列大小 |
```
Comment (GregoryIan) [inline dev/how-to/monitor/tidb-binlog.md]: Queue？

File: v2.1/how-to/monitor/tidb-binlog.md [hunk 2]
```diff
@@ -30,10 +30,12 @@ category: reference
 | Pump Handle TSO | 记录 Drainer 从各个 Pump 获取到的 binlog 的最大 TSO 对应的时间 | | Pull Binlog QPS by Pump NodeID | Drainer 从每个 Pump 获取 binlog 的 QPS |
 | 95% Binlog Reach Duration By Pump | 记录 binlog 从写入 Pump 到被 Drainer 获取到这个过程的延迟时间 |
 | Error By Type | Drainer 遇到的 error 数量，按照 error 的类型进行统计 |
+| SQL Query Time| Drainer 在下游执行 SQL 的耗时 |
 | Drainer Event | 各种类型 event 的数量，event 包括 ddl、insert、delete、update、flush、savepoint |
 | Execute Time | 在下游执行 SQL 语句或写数据所消耗的时间 |
```
Comment (WangXiangUSTC) [inline v2.1/how-to/monitor/tidb-binlog.md]: 这个是执行一个事务消耗的时间？应该废弃了吧

Comment (july2993) [reply v2.1/how-to/monitor/tidb-binlog.md]: 2.1 的没改，统一改下描述把

**Comment by GregoryIan** [comment]
LGTM

**Pull request merged**